FE/src/components/Admin.jsx

```diff
@@ -1,8 +1,301 @@
+                // const result = await response.json()             
+                if (token !== null) {//need an elseif to crosscheck admin
+                    setReviewedRecipesList(dummyReviewedRecipes);
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }allReviewedRecipesCheck();
+    }, [])
+
+    useEffect(()=>{
+        async function allTagsCheck () {
+            try {
+                // const response = await fetch(`${API}tags, {
+                //     method: "GET",
+                //     headers: {
+                //         "Content-Type":"application/json",
+                //         "Authorization": `Bearer ${token}`,
+                //     },
+                // });
+                // const result = await response.json()             
+                if (token !== null) {//need an elseif to crosscheck admin
+                    setTagsList(dummyTags);
+                    setFail(false);
+                    setSuccess(true);
+                    // setUserBio(true);
+                    // setUserForm(false);
+                } else {
+                    setFail(true);
+                    setSuccess(false);
+                }
+
+            } catch (error) {
+                setError(error.message);
+                console.log( error );
+            }
+        }allTagsCheck();
+    }, [])
 
     return (
-        <><p>This is the Admin page for admin specific tasks</p></>
+        <><p>This is the Admin page for admin specific tasks</p>
+        <div className = 'wrapper'>
+            <div className='adminNav'>
+                <button onClick={() => {
+                    setAllUsers(true);
+                    setReviewedRecipes(false);
+                    setrecipeTags(false);
+                }}>All Users</button>
+                <button onClick={() => {
+                    setAllUsers(false);
+                    setReviewedRecipes(true);
+                    setrecipeTags(false);
+                }}>Reviewed Recipes</button>
+                <button onClick={() => {
+                    setAllUsers(false);
+                    setReviewedRecipes(false);
+                    setrecipeTags(true);
+                }}>Recipe Tags</button>
+            </div>
+            {error && <p>{error}</p>}
+            {fail && <div className='fail'><h2>Please Sign In</h2><br/> <button type='login' onClick={signIn}>Log In</button></div>}
+            {success &&
+            <div>
+                {allUsers &&
+                <div className = 'allUsers'>
+                    <h2>All Users</h2>
+                    {usersList.map((user)=> {
```

Comment:
When we eventually refactor this into subcomponents, this and the regular user page will be a whole lot cleaner, and able to reuse a lot of the same stuff.